From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shobek Attupurath <sattupurath@nvidia.com>
Date: Tue, 5 Nov 2024 02:24:38 +0000
Subject: [PATCH 004/243] NVIDIA: SAUCE: bt_usb: Disable RTL8852CE upstream
 driver

Disable RTL8852CE upstream bluetooth driver detection to
support downstream driver

Bug 4915378

Change-Id: Ia9b1523311c12b86b0d69bc37dffed3c4f469ac9
Signed-off-by: Shobek Attupurath <sattupurath@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/3rdparty/canonical/linux-jammy/+/3251663
(cherry picked from commit 987407ca2d2534c53d90445d0eeaae5e8847cacf)
Reviewed-on: https://git-master.nvidia.com/r/c/3rdparty/canonical/linux-noble/+/3249199
Reviewed-by: Ashutosh Jha <ajha@nvidia.com>
GVS: buildbot_gerritrpt <buildbot_gerritrpt@nvidia.com>
(cherry picked from commit 4fd847e5fc26bf6386537abf11ed797162418cb4)

diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index d5b8b618f368..8c1c15a94512 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -560,8 +560,7 @@ static const struct usb_device_id quirks_table[] = {
 						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x13d3, 0x3587), .driver_info = BTUSB_REALTEK |
 						     BTUSB_WIDEBAND_SPEECH },
-	{ USB_DEVICE(0x13d3, 0x3586), .driver_info = BTUSB_REALTEK |
-						     BTUSB_WIDEBAND_SPEECH },
+	{ USB_DEVICE(0x13d3, 0x3586), .driver_info = BTUSB_IGNORE },
 	{ USB_DEVICE(0x13d3, 0x3592), .driver_info = BTUSB_REALTEK |
 						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x13d3, 0x3612), .driver_info = BTUSB_REALTEK |
-- 
2.43.0

